### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by subhanibabu**
Please review HomeTask3.

### CODE REVIEW

File: HomeTask3/Controller.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+        view.on('addButtonClicked', idx => this.SelectCategory(idx));
+        view.on('validateEmailButtonClicked', () => this.ValidateEmail());
+        view.on('addOpenPopUpbuttonclick', value => this.OpenPopUp(value));
+        view.on('ClosePopUpButtonClick', () => this.ClosePopUp());
+    }
+
+    SelectCategory(value) {
+        if (value == 0) {
+            for (var i = 1; i < 11; i++) {
```
Comment (AshisSarthak) [inline HomeTask3/Controller.js]: have meaning full variable name

File: HomeTask3/Controller.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        this._view = view;
+
+        view.on('addButtonClicked', idx => this.SelectCategory(idx));
+        view.on('validateEmailButtonClicked', () => this.ValidateEmail());
+        view.on('addOpenPopUpbuttonclick', value => this.OpenPopUp(value));
+        view.on('ClosePopUpButtonClick', () => this.ClosePopUp());
+    }
+
+    SelectCategory(value) {
+        if (value == 0) {
```
Comment (AshisSarthak) [inline HomeTask3/Controller.js]: Use === instead of ==

File: HomeTask3/HomeTask3.css [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    top: 0 !important;
+    width: 100% !important;
+    height: 100% !important;
+    overflow: auto !important;
+    background-color: rgb(0,0,0) !important;
+    background-color: rgba(0,0,0,0.4) !important;
+}
+
+.modal-content {
+    background-color: #fefefe !important;
```
Comment (AshisSarthak) [inline HomeTask3/HomeTask3.css]: Avoid using !important

File: HomeTask3/EventDispatcher.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class EventEmitter {
```
Comment (AshisSarthak) [inline HomeTask3/EventDispatcher.js]: I am not sure about this event Emitter class
If you intend to use customEvents you can do that via window object. This class seems redundant for me

File: HomeTask3/Controller.js (hunk 1 above)
Comment (subhanibabu) [reply HomeTask3/Controller.js]: Changed names.

File: HomeTask3/Controller.js (hunk 2 above)
Comment (subhanibabu) [reply HomeTask3/Controller.js]: Used ===

File: HomeTask3/HomeTask3.css (hunk 3 above)
Comment (subhanibabu) [reply HomeTask3/HomeTask3.css]: Removed !important

File: HomeTask3/EventDispatcher.js (hunk 4 above)
Comment (subhanibabu) [reply HomeTask3/EventDispatcher.js]: Removed event emitter class.

**Pull request merged**